src/main/java/com/vestmark/nexus/plugin/maven/MavenResource.java

```diff
@@ -184,6 +184,11 @@ public Response download(
       response.header(HttpHeaders.CONTENT_DISPOSITION, String.format("attachment;filename=\"%s\"", fileName));
       return response.build();
     }
+    catch (RuntimeException e) {
+      log.error("storageTx response exception: {}", e.toString());
```

Comment:
This is still wrong.  The `{}` need to go, and the second argument needs to be `e`.

Reply:
Changing this to:
```java
log.error("storageTx response exception: ", e);
```
Might give us more information.

Reply:
My latest commit has that change.